sbi/inference/snpe/snpe_c.py

```diff
@@ -141,11 +141,9 @@ def _get_log_prob_proposal_posterior(
         unnormalized_log_prob_proposal_posterior = log_prob_posterior - log_prob_prior
 
         # Normalize proposal posterior across discrete set of atoms.
-        log_prob_proposal_posterior = self.calibration_kernel(
-            x
-        ) * unnormalized_log_prob_proposal_posterior[:, 0] - torch.logsumexp(
-            unnormalized_log_prob_proposal_posterior, dim=-1
-        )
+        log_prob_proposal_posterior = unnormalized_log_prob_proposal_posterior[
```

Comment:
Black is here extremely annoying, to the point of unreadability. Maybe shorten the variable name?